### PR TITLE
Tendrils are now structure based [READY]

### PR DESCRIPTION
**Pull request by 4dplanner**
Tendrils are structure based, and spawners are a component.

### CODE REVIEW

**Comment by 4dplanner** [comment]
note: crushers will need some snowflake code

File: code/modules/ruins/objects_and_mobs/ash_walker_den.dm [hunk 1]
```diff
@@ -1,26 +1,32 @@
...
 	icon_state = "ash_walker_nest"
-	icon_living = "ash_walker_nest"
-	icon_dead = "ash_walker_nest"
-	faction = list("ashwalker")
-	health = 200
-	maxHealth = 200
-	loot = list(/obj/effect/collapse)
+	var/faction = list("ashwalker")
+	max_integrity = 200
 	var/meat_counter = 6
```
Comment (ShizCalev) [inline code/modules/ruins/objects_and_mobs/ash_walker_den.dm]: Move the new vars below ones defined by the parent obj

**Review by ShizCalev — DISMISSED** [review]

**Comment by Tlaltecuhtli** [comment]
well you can do something like fireaxes having 20000 damage against windows

**Review by ChangelingRain — APPROVED** [review]

**Comment by Haevacht** [comment]
Crushers can't currently mark tendrils anyway, and they have good melee damage. As long as bashing it works like bashing a window, it'll be fine.

**Comment by 4dplanner** [comment]
> Crushers can't currently mark tendrils anyway, and they have good melee damage. As long as bashing it works like bashing a window, it'll be fine.

Yeah, I realised that and that's why it's marked ready

**Comment by 4dplanner** [comment]
> Crushers can't currently mark tendrils anyway, and they have good melee damage. As long as bashing it works like bashing a window, it'll be fine.

Smashing it with melee damage and pka were both tested, and they certainly seemed to work fine.

**Comment by Arianya** [comment]
Make sure this doesn't cause oddities with the custom portal storm spawner (Secrets panel), since tendrils are one of the things that you can portal with them.

**Pull request merged**